src/index.html.ejs

```diff
@@ -1,176 +0,0 @@
-
-  // Initialize test instance of game, exposed to window for debugging.
-  var gameController = new GameController({
-    Phaser: window.Phaser,
-    containerId: 'phaser-game',
-    assetRoot: 'assets/',
-    audioPlayer: new Sounds(),
-    debug: true,
-    earlyLoadAssetPacks: testLevelToLoad.earlyLoadAssetPacks,
-    earlyLoadNiceToHaveAssetPacks: testLevelToLoad.earlyLoadNiceToHaveAssetPacks,
-    afterAssetsLoaded: () => {
-      gameController.codeOrgAPI.startAttempt(() => {});
-    },
-  });
-
-  gameController.loadLevel(testLevelToLoad);
-
-  $levelselect = $('#level-load');
-  Object.keys(demoLevels).forEach(key => {
-    $levelselect.append($('<option/>', {text: key, selected: key === levelParam}));
-  });
-
-  $levelselect.on('change', () => {
-    location.search = `level=${$levelselect.val()}`;
-  });
-
-  $('input[type=range]').on('input', function () {
-    $("#speed-display").html('Speed: ' + $(this).val() + 'x');
-    gameController.game.time.slowMotion = 1.5 / parseFloat($(this).val(), 10);
-  });
-
-  $('#reset-button').click(function () {
-    gameController.codeOrgAPI.resetAttempt();
-    gameController.codeOrgAPI.startAttempt(() => {});
-  });
-
-  window.addEventListener('keydown', e => {
-    if (e.target !== document.body) {
-      e.preventDefault();
-    }
-    e.stopImmediatePropagation();
-
-    var target = $('input[name=target]:checked').val();
-    var instance = target === 'Player' ? gameController.player : gameController.agent;
-
-    switch (e.keyCode) {
-      case 8:
-      case 46:
-        gameController.codeOrgAPI.destroyBlock(null, target);
-        break;
-      case 13:
-        gameController.codeOrgAPI.placeInFront(null, $('#block-type').val(), target);
-        break;
-      case 16:
-        $('input[name=target]:not(:checked)').prop('checked', true);
-        break;
-      case 38:
-      case 87:
-        instance.movementState = 0;
-        instance.updateMovement();
-        break;
-      case 40:
-      case 83:
-        instance.movementState = 2;
-        instance.updateMovement();
-        break;
-      case 37:
-      case 65:
-        instance.movementState = 3;
-        instance.updateMovement();
-        break;
-      case 39:
-      case 68:
-        instance.movementState = 1;
-        instance.updateMovement();
-        break;
-    }
-  }, true);
-
-  window.addEventListener('keyup', e => {
-    e.stopImmediatePropagation();
-
-    var target = $('input[name=target]:checked').val();
-    var instance = target === 'Player' ? gameController.player : gameController.agent;
-
-    instance.movementState = -1;
-    instance.updateMovement();
-  }, true);
-
-  window.gameController = gameController;
-</script>
-
-<% if (liveReloadPort) { %>
-<script>
-  document.write(
-    '<script src="//'
-    + window.location.hostname
-    + ':<%= liveReloadPort %>/livereload.js"><\/script>'
-  );
-</script>
-<% } %>
-
-</body>
-</html>
```

Comment:
Nit: this cleanup could be pre-flighted in a separate PR to reduce the diff.

Reply:
I'm actually not sure it could; we're able to move this file from `index.html.ejs` to `index.html` because we are able to get rid of the `liveReloadPort` stuff here at the end, but we're only able to do that because webpack dev server provides it for us automatically

Reply:
Ah, my bad!